index.html

```diff
@@ -418,8 +418,8 @@ <h2 style="">Contact</h2>
 		<hr color="white">
 
     <div>
-			<i class="fa fa-info-circle" aria-hidden="true"></i> PSLab | FOSSASIA
-		</div>
+			<i class="fa fa-info-circle" aria-hidden="true"></i> PSLab | <a href="http://fossasia.org">FOSSASIA</a> <a href="privacy_policy.html">| Privacy</a> 
```

Comment:
Hi @viveksb007 ! What if we exclude the bar line from link and start the anchor tag after it?